db/user_test.go

```diff
@@ -14,6 +15,83 @@ type SyncNewUsersTestCase struct {
     sendEmails bool
 }
 
+type ResolveUsersTestCase struct {
```

Comment:
Should not be exported.

Edit:
See lower comments, this actually doesn't need to exist.
You can just use anonymous structs like in most other test cases.
E.g., https://github.com/edulinq/autograder-server/blob/main/db/submission_test.go#L13